Added three dots animated loader

prahe184 opened the pull request:
#Description:
I have added 3 dots animated loader

Uploading Screen Recording 2025-05-27 190014.mp4…



Closes #126

ShreyTanna29 commented:
@prahe184 1. color of loader should be black in landing page 
                      2. remove the text "get Started" when loading
                      3. keep laoding spinner component as it is, make a new component file Loading dots
                      4. remove the changes made in loader.tsx , keep it as it is.
                      
![Screenshot from 2025-05-27 18-59-10](https://github.com/user-attachments/assets/7b3b3d83-64e2-473e-a0d2-a85a5285c9d8)

prahe184 commented:
> @prahe184 1. color of loader should be black in landing page 2. remove the text "get Started" when loading 3. keep laoding spinner component as it is, make a new component file Loading dots 4. remove the changes made in loader.tsx , keep it as it is.
> 
> ![Screenshot from 2025-05-27 18-59-10](https://private-user-images.githubusercontent.com/147817986/447947289-7b3b3d83-64e2-473e-a0d2-a85a5285c9d8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDg0MzI1NTEsIm5iZiI6MTc0ODQzMjI1MSwicGF0aCI6Ii8xNDc4MTc5ODYvNDQ3OTQ3Mjg5LTdiM2IzZDgzLTY0ZTItNDczZS1hMGQyLWE4NWE1Mjg1YzlkOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUyOFQxMTM3MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNzQ2NzJmNmE3NDJhOWIzZTFkZDIzMGJmOGUyNDMzMzJjN2I5MTgwYzFiZWFjZTNkZGI0NTE3ZjY3ZmZmZTViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.4hoF2gXhrd-1hC1pB3Ovx08sCBoInOuUq-AU_83BANg)





https://github.com/user-attachments/assets/8a091889-14ff-45a4-9950-eace31
Hey @ShreyTanna29, I fixed the requirements listed above. Once review it. I fixed it yesterday itself but stuck with small error...
